### PR TITLE
Inventory & Inventory Details

### PR DESCRIPTION
**Pull request by inphomercial**
Adding description to items, adding screenSubSwitch method to allow for passed data. Added inventory details screen, and basic interactions.

### CODE REVIEW

File: src/maps/overworld.js [hunk 1]
```diff
@@ -46,11 +46,16 @@ class Overworld extends Map {
...
 		player.getComponent("Inventory").inventory.push(sword1);
-		player.getComponent("Inventory").inventory.push(sword2);
-		player.getComponent("Slots").slots.hand = dagger;
+		player.getComponent("Inventory").inventory.push(dagger);
+		player.getComponent("Inventory").inventory.push(dagger);
+		player.getComponent("Inventory").inventory.push(dagger);
+		player.getComponent("Inventory").inventory.push(dagger);
+		player.getComponent("Inventory").inventory.push(dagger);
+		player.getComponent("Inventory").inventory.push(dagger);
+		player.getComponent("Inventory").inventory.push(dagger);
```
Comment (lallenlowe) [inline src/maps/overworld.js]: Why are you pushing hella daggers?

File: src/screens/inventory.js [hunk 2]
```diff
@@ -35,19 +36,37 @@ Lootr.Screens.Inventory = {
...
 				this._selectedIndex--;
+				this._selectedItem = this._inventory[this._selectedIndex - this._startingPostition];
+
 				break;
 
 			case ROT.KEYS.VK_ESCAPE:
 				Lootr.switchScreen(Lootr.Screens.Play);
 				break;
+			
+			case ROT.KEYS.VK_RETURN:
```
Comment (lallenlowe) [inline src/screens/inventory.js]: do Enter as well as Return.

Comment (inphomercial) [reply src/screens/inventory.js]: good idea!

File: src/maps/overworld.js (hunk 1 above)
Comment (inphomercial) [reply src/maps/overworld.js]: just to test the inventory with many items 🗡

**Pull request merged**